File: src/ProjectR/content/scripts/spells/ForbiddenSpell74.lua
function GetTargetType()
  return Enemies
end

function GetName()
  return "Forbidden Spell #74"
end

function GetDescription()
  return "Through some foul magic this spell's damage is reduced by DEF.\nTargets all enemies."
end

function IsSupportSpell()
    return false
end

function GetMPCost()
    return .96
end

function GetDelay()
  return .35
end

function GetSpellType()
  return Magical
end

function GetMasteries()
  return {  }
end

function SpellEffect()
  local damage = 3.5 * aMD - .875 * dDEF
  d:TakeDamage(damage)
end

File: src/ProjectR.Scripting/content/scripts/spells/ForbiddenSpell74.cs
using System.Collections.Generic;
using ProjectR.Interfaces;
using ProjectR.Interfaces.Model;
using ProjectR.Interfaces.Model.Stats;

namespace ProjectR.Scripting.Spells
{
    public class ForbiddenSpell74 : SpellScriptBase
    {
        public override string Name { get { return "Forbidden Spell #74"; } }
        public override string Description { get { return "Through some foul magic this spell's damage is reduced by DEF.\nTargets all enemies."; } }

        public override TargetType TargetType { get { return TargetType.Enemies; } }
        public override IList<EleMastery> Masteries { get { return new EleMastery[] { }; } }
        public override SpellType SpellType { get { return SpellType.Magical; } }
        public override bool IsSupportSpell { get { return false; } }

        public override double MPCost { get { return 96; } }
        public override double Delay { get { return .35; } }

        public override void SpellEffect(ICharacter caster, ICharacter target)
        {
            var damage = 3.5 * cMD - .75 * tDEF;
            DealDamage(damage);
        }
    }
}